### PR TITLE
[UPD] Use connector unit

### PR DESCRIPTION
**Pull request by ehdem**
Readapt the code with connector unit

### CODE REVIEW

File: cmis/cmis_backend.py [hunk 1]
```diff
@@ -37,37 +39,40 @@ class cmis_backend(orm.Model):
             lambda self, *a, **kw: self._select_versions(*a, **kw),
             string='Version',
             required=True,
+            help="Version",
```
Comment (guewen) [inline cmis/cmis_backend.py]: Not sure it helps to write the same string than in `string`.

File: cmis/unit/binder.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                     ])
+            if erp_id:
+                record_id = erp_id[0]
+            else:
+                return None
+
+        redmine_record = self.session.read(
+            self.model._name, record_id, ['dms_id'])
+
+        return redmine_record['redmine_id']
```
Comment (guewen) [inline cmis/unit/binder.py]: s/redmine_id/dms_id/

File: cmis/unit/binder.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+        :type binding_id: int
+        """
+        # avoid to trigger the export when we modify the `redmine_id`
+        context = self.session.context.copy()
+        context['connector_no_export'] = True
+        now_fmt = datetime.now().strftime(DEFAULT_SERVER_DATETIME_FORMAT)
+
+        self.environment.model.write(
+            self.session.cr, self.session.uid, binding_id, {
+                'redmine_id': ustr(external_id),
```
Comment (guewen) [inline cmis/unit/binder.py]: s/redmine_id/dms_id/

File: cmis/cmis_backend.py (hunk 1 above)
Comment (guewen) [inline cmis/cmis_backend.py]: I was talking about the text in `help` which is the same than in `string`

Comment (ehdem) [inline cmis/cmis_backend.py]: @guewen: it is just the cmis version. Maybe, it is better to put "Cmis version" in the help

Comment (guewen) [inline cmis/cmis_backend.py]: My opinion: remove the help if there is nothing useful to say (that's the same thing for all the other fields below).

Comment (ehdem) [inline cmis/cmis_backend.py]: @guewen : alright

File: cmis/unit/binder.py [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+#    along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from openerp.addons.connector.connector import Binder
+from ..backend import cmis
+
+from datetime import datetime
+from tools import ustr
```
Comment (lmignon) [inline cmis/unit/binder.py]: from openerp.tools import ustr

File: cmis/cmis_backend.py [hunk 5]
```diff
@@ -22,9 +22,11 @@
 
 from openerp.osv import orm, fields
 from openerp.tools.translate import _
-from cmislib.model import CmisClient
+from openerp.addons.connector.connector import Environment
+from openerp.addons.connector.session import ConnectorSession
+from openerp.addons.cmis.unit.backend_adapter import CmisAdapter
+from .unit.backend_adapter import CmisAdapter
```
Comment (lmignon) [inline cmis/cmis_backend.py:27]: Import redefinition: CmisAdapter

**Comment by lmignon** [comment]
Tested on 8.0 and it works fine as it is  (Once the commented import is fixed)

**Comment by max3903** [comment]
:+1:

**Pull request merged**